lib/ui/myplot/PlotList.dart

```diff
@@ -57,74 +53,72 @@ class _DefaultStyle implements PlotListStyle {
   final TextStyle titleTextStyle = const TextStyle(
       fontSize: 27, fontWeight: FontWeight.bold, color: Color(0xFF000000));
 
-  final String errorText = "Something went wrong while loading data";
-  final String buttonText = "Add Another Crop";
-  final String errorButtonText = "Retry";
-
   const _DefaultStyle();
 }
 
-class PlotList extends StatefulWidget {
-  @override
-  State<StatefulWidget> createState() {
-    return _MyPlotState();
-  }
-}
+class PlotList extends StatelessWidget {
+  final ViewModelProvider<PlotListViewModel> _viewModelProvider;
+
+  const PlotList({Key key, ViewModelProvider<PlotListViewModel> provider}) : this._viewModelProvider = provider, super(key: key);
 
-class _MyPlotState extends State<PlotList> {
   @override
   Widget build(BuildContext context,
-      {PlotListStyle plotStyle = const _DefaultStyle()}) {
-    return Scaffold(
-      body: StoreConnector<AppState, MyPlotViewModel>(
-          onInit: (store) => store.dispatch(FetchCropListAction()),
-          builder: (_, viewModel) => _buildBody(context, viewModel, plotStyle),
-          converter: (store) => MyPlotViewModel.fromStore(store)),
-    );
+      {PlotListStyle style = const _DefaultStyle()}) {
```

Comment:
This should be here? Or should be inside the PlotList constructor?

Reply:
yeah should be in the constructor 👍